packages/ngtools/webpack/README.md

```diff
@@ -1,6 +1,9 @@
 # Angular Compiler Webpack Plugin
 
 Webpack 5.x plugin for the Angular Ahead-of-Time compiler. The plugin also supports Angular JIT mode.
+When this plugin is used outside of the Angular CLI, the Ivy linker will also be needed to support
+the usage of Angular libraries. An example configuration of the Babel-based Ivy linker is provided
+in the linker section. For additional information regarding the linker, please see: https://v13.angular.io/guide/creating-libraries#consuming-partial-ivy-code-outside-the-angular-cli
```

Comment:
Maybe remove the v13 from the url since it will be release as stable later on today?

Reply:
Was planning to do some cleanup to the structure/formatting of the option list as well. I can remove the v13 part then also since those further changes will be after the release.